sdk/dotnet/Pulumi/Serialization/Deserializer.cs

```diff
@@ -75,6 +75,13 @@ private static OutputData<double> DeserializerDouble(Value value)
 
                     foreach (var (key, element) in v.StructValue.Fields)
                     {
+                        // Unilaterally skip properties considered internal by the Pulumi engine.
```

Comment:
I don't think that this is the right approach. Why are we trying to deserialize properties that we don't need at all?

Reply:
Where do we try to deserialize them?